<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiVA_detector_qtClass</class>
 <widget class="QMainWindow" name="DiVA_detector_qtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DiVA_detector_qt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushIniciar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushPrivacidad">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>380</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Privacy</string>
    </property>
   </widget>
   <widget class="QLabel" name="display">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>531</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>                                                                                  Video</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="VPULogo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>101</width>
      <height>61</height>
     </size>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/DiVA_detector_qt/Resources/Logos/eventVideo_logo.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="EPSLogo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>480</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>121</width>
      <height>41</height>
     </size>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/DiVA_detector_qt/Resources/Logos/MinecoLogo-200.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="UAMLogo">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>470</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/DiVA_detector_qt/Resources/Logos/VPU_logo-200.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="MinEcoLogo">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>470</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/DiVA_detector_qt/Resources/Logos/LogoOficial-EPS-200.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushCam">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>380</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Camara</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushDetener">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Detener</string>
    </property>
   </widget>
   <widget class="QLabel" name="HAVideoLogo">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>450</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/DiVA_detector_qt/Resources/Logos/LogoOficial-EPS-200.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="camName">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>120</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Camara</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>893</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHerramientas">
    <property name="title">
     <string>Herramientas</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSeleccion_Parametros"/>
    <addaction name="actionSeleccion_de_camara"/>
    <addaction name="actionSeleccion_de_region_de_analisis"/>
    <addaction name="actionModo_compacto"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAyuda_2"/>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuHerramientas"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionCamera"/>
   <addaction name="actionPlay"/>
   <addaction name="actionStop"/>
   <addaction name="actionConfiguration"/>
   <addaction name="actionAyuda_3"/>
  </widget>
  <action name="actionAyuda">
   <property name="text">
    <string>Ayuda</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="icon">
    <iconset resource="diva_detector_qt.qrc">
     <normaloff>:/DiVA_detector_qt/Resources/Iconos/cam.png</normaloff>:/DiVA_detector_qt/Resources/Iconos/cam.png</iconset>
   </property>
   <property name="text">
    <string>camera</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="diva_detector_qt.qrc">
     <normaloff>:/DiVA_detector_qt/Resources/Iconos/Play.png</normaloff>:/DiVA_detector_qt/Resources/Iconos/Play.png</iconset>
   </property>
   <property name="text">
    <string>play</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="diva_detector_qt.qrc">
     <normaloff>:/DiVA_detector_qt/Resources/Iconos/stop.png</normaloff>:/DiVA_detector_qt/Resources/Iconos/stop.png</iconset>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="icon">
    <iconset resource="diva_detector_qt.qrc">
     <normaloff>:/DiVA_detector_qt/Resources/Iconos/engranaje.png</normaloff>:/DiVA_detector_qt/Resources/Iconos/engranaje.png</iconset>
   </property>
   <property name="text">
    <string>configuration</string>
   </property>
  </action>
  <action name="actionAyuda_3">
   <property name="icon">
    <iconset resource="diva_detector_qt.qrc">
     <normaloff>:/DiVA_detector_qt/Resources/Iconos/ayda.png</normaloff>:/DiVA_detector_qt/Resources/Iconos/ayda.png</iconset>
   </property>
   <property name="text">
    <string>ayuda</string>
   </property>
   <property name="toolTip">
    <string>ayuda</string>
   </property>
  </action>
  <action name="actionSeleccion_Parametros">
   <property name="text">
    <string>Configuracion de parametros</string>
   </property>
  </action>
  <action name="actionSeleccion_de_camara">
   <property name="text">
    <string>Seleccion de camara</string>
   </property>
  </action>
  <action name="actionSeleccion_de_region_de_analisis">
   <property name="text">
    <string>Seleccion de region de analisis</string>
   </property>
  </action>
  <action name="actionModo_compacto">
   <property name="text">
    <string>Modo compacto</string>
   </property>
  </action>
  <action name="actionAyuda_2">
   <property name="text">
    <string>Ayuda</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="diva_detector_qt.qrc"/>
 </resources>
 <connections/>
</ui>
